ibis/backends/bigquery/registry.py

```diff
@@ -330,58 +324,65 @@ def _log(translator, op):
     return f"log({arg_formatted}, {base_formatted})"
 
 
-def _literal(translator, op):
+def _lit(val) -> str:
```

Comment:
Small not - could you come up with a better name here? Having two adjacent functions named `_lit` and `_literal` it's not immediately clear (without reading them) what the difference is.

Reply:
Yep, on it!

Reply:
Renamed to `_sg_literal`